src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -276,6 +278,7 @@ public KinesisClientLibConfiguration(String applicationName,
     public KinesisClientLibConfiguration(String applicationName,
             String streamName,
             String kinesisEndpoint,
+            String dynamoDBEndpoint,
```

Comment:
Since this changes a public constructor, could you please add a constructor override?  This could break people's code due to the addition of a new constructor parameter.